local-volume/provisioner/pkg/controller/controller.go

```diff
@@ -53,7 +53,7 @@ func StartLocalController(client *kubernetes.Clientset, ptable deleter.ProcTable
 	// We choose a random resync period between MinResyncPeriod and 2 *
 	// MinResyncPeriod, so that local provisioners deployed on multiple nodes
 	// at same time don't list the apiserver simultaneously.
-	resyncPeriod := time.Duration(float64(config.MinResyncPeriod.Nanoseconds()) * (1 + rand.Float64()))
+	resyncPeriod := time.Duration(config.MinResyncPeriod.Seconds()*(1+rand.Float64())) * time.Second
```

Comment:
I think if you don't cast it to float64 like before, then you don't need to multiple by time.Second at the end.

Reply:
`Seconds()` method divides it by time.Second to get number of seconds, so we need to multiple by time.Second at the end. If you prefer not to multiple by time.Second, we need to calculate it using nanoseconds. Actually time.Duration is int64 in nanoseconds.

Demo: https://play.golang.org/p/taL1ywwQ1wt